content/blog/gifts-q3-2020.md

```diff
@@ -0,0 +1,66 @@
+
+Without further ado, here are the recipients:
+
+### Gold
+[`~radbur-sivmus`](https://github.com/tylershuster) has more or less become an unofficial member of the interface team by tackling an absolutely [immense](https://github.com/urbit/urbit/pulls?q=is%3Apr+author%3Atylershuster+) amount of Landscape-related work.
+
+### Silver
+- [`~lagrev-nocfep`](https://github.com/davis68): In addition to being highly active throughout nearly all of Urbit, Neal has created a first of its kind graduate-level course focused on Urbit at the University of Illinois: [CS 498MC](https://relate.cs.illinois.edu/course/cs498mc-fa20/).
+- [`~littel-wolfur`](https://github.com/ryjm): Jake maintains the premier development group in Urbit, The Forge, and has led the way for other developers by authoring [srrs](https://github.com/ryjm/srrs), one of the more popular community-developed applications on Urbit.
+- [`~botter-nidnul`](https://github.com/botter-nidnul): ~botter has been instrumental in furthering [Urbit on ARM](https://botter-nidnul.github.io/), has made contributions across Vere and our documentation, and been highly active in onboarding newcomers on our Discord server.
+- [`~timluc-miptev`](https://github.com/timlucmiptev): ~timluc is highly engaged in many groups and the maintainer of some, wrote the [Nock for Everyday Coders](https://blog.timlucmiptev.space/part1.html) guide, and has gone far above and beyond in teaching the community about developing Landscape applications.
+- [`~lanrus-rinfep`](https://github.com/dclelland): Daniel has done extensive development on iOS by writing Swift libraries for interacting with Urbit, and employed all of these towards the development of an [iOS client for chat](https://github.com/dclelland/UrsusChat).
+- [`~minder-folden`](https://twitter.com/the_pritchard): Jonathan is a prolific member of the community. His group *The Sanctorium* is always active, he's always around to help newcomers to the system, is an active evangelist on Twitter, and a creator of much excellent content.
+
+### Bronze
+- [`~watter-parter`](https://github.com/lukechampine): Luke has made a number of [contributions](https://github.com/urbit/urbit/pulls?q=is%3Apr+is%3Aclosed+author%3Alukechampine) to Arvo, and wrote a great [introductory guide](https://github.com/lukechampine/rote) to developing Gall agents.
+- [`~naltyc-wornes`](https://github.com/JohnELester): ~naltyc runs the popular group “Smol Computers” both inside and outside of Urbit where discussions ensue about various forms of computing on small devices like Android and ARM, and actively evangelizes the project.
+- [`~nartes-fasrum`](https://twitter.com/nartesfasrum): ~nart evangelizes Urbit on Twitter by creating artwork, and maintains the popular food-centric group “Urbytes.”
+- [`~fonnyx-nopmer`](https://twitter.com/fonnyx_nopmer): ~fonnyx maintains an active presence throughout many groups on Urbit and runs the first Urbit-centric podcast, Hooked on fonnyx.
```

Comment:
`~fonnyx`

Reply:
Might be worth linking to twitch.tv/fonnyx for the podcast and put it in quotes.